doc/start/hardware-recommendations.rst

```diff
@@ -120,16 +121,17 @@ multiple daemons against a single drive can slow performance considerably.
 Hard Disk Drives
 ----------------
 
-OSDs should have plenty of hard disk drive space for object data. We recommend a
-minimum hard disk drive size of 1 terabyte. Consider the cost-per-gigabyte
+OSDs should have plenty of hard disk drive space for object data. We recommend
+a minimum hard disk drive size of 1 terabyte. Consider the cost-per-gigabyte
 advantage of larger disks. We recommend dividing the price of the hard disk
 drive by the number of gigabytes to arrive at a cost per gigabyte, because
 larger drives may have a significant impact on the cost-per-gigabyte. For
 example, a 1 terabyte hard disk priced at $75.00 has a cost of $0.07 per
-gigabyte (i.e., $75 / 1024 = 0.0732). By contrast, a 3 terabyte hard disk priced
-at $150.00 has a cost of $0.05 per gigabyte (i.e., $150 / 3072 = 0.0488). In the
-foregoing example, using the 1 terabyte disks would generally increase the cost
-per gigabyte by 40%--rendering your cluster substantially less cost efficient.
+gigabyte (i.e., $75 / 1024 = 0.0732). By contrast, a 3 terabyte hard disk
+priced at $150.00 has a cost of $0.05 per gigabyte (i.e., $150 / 3072 =
+0.0488). In the foregoing example, using the 1 terabyte disks would generally
+increase the cost per gigabyte by 40%--rendering your cluster substantially
+less cost efficient.
```

Comment:
With spinners, though, the SATA or even SAS interface increasingly becomes a bottleneck at larger capacities.  AIUI RHCS advises a max 10TB capacity for this reason.  Maybe we should include a note that larger capacities can be bottlenecked?

Since this touches on TCO, I might also suggest including a link to https://www.snia.org/forums/cmsi/programs/TCOcalc

Reply:
I've added a tip that mentions this, and I put a link to the TCO calculator here and another place in this document.

Reply:
I see the link at the bottom, but not within the text.  Am I missing it?

Reply:
Sorry about that. It's present in my local file system and it works correctly, but time and circumstance have resulted in several hours of editing more that forestalled this change's commitment and pushing to origin. So: stand by.